src/components/portfolio/EditDialog.tsx

```diff
@@ -287,6 +287,21 @@ function reducer(state: State, action: Actions): State {
 }
 
 const useStyles = makeStyles((theme) => ({
+  dialog: {
+    '& .MuiDialog-container': {
+      minWidth: '15rem',
+    },
```

Comment:
What is the purpose of this `minWidth`?

Reply:
This is needed so that even at the smallest possible resize all buttons of the dialog are still usable. Functionality is never lost no matter the size of the window.